pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -65,8 +64,8 @@ type Bootstrapper struct {
 }
 
 // NewBootstrapper creates a new kubeadm.Bootstrapper
-func NewBootstrapper(api libmachine.API) (*Bootstrapper, error) {
-	name := viper.GetString(config.MachineProfile)
+func NewBootstrapper(api libmachine.API, cc config.ClusterConfig, n config.Node) (*Bootstrapper, error) {
```

Comment:
Please add a TODO(#xxx) to remove node as an argument, and/or clarify in the docstring what the node is doing here.